examples/compound/README.md

```diff
@@ -1,2 +1,288 @@
+defined
+in `.abi.go`) into protobuf format, mapping all data to their corresponding fields. You are now done with the
+contract-handling part of the connector!
+
+#### 4. Create the connector runtime files
+
+Now you will create the files to allow your connector to call the Nakji connector library and use it to instantiate and
+run your connector. These include all the files in the <connector_name>/ directory. You can start by copying the files
+from `compound/` into `<connector_name>/`. Then you need to modify each file to fit your connector.
+
+* `contract.go`
+* `local.yaml`
+* `manifest.yaml`
+* `types.go`
+* `<connector_name>.go`
+
+`contract.go`: This defines the functions to retrieve the Ethereum ABI from the contract address and save it to a field
+in
+our connector struct. You can copy this file directly from `compound`.
+
+`local.yaml`: This is the configuration file containing the rpc urls which are needed to retrieve data from the
+blockchain. You’ll need it to run and test your connector locally.
+
+`manifest.yaml`: This contains info about the connector name, author, version etc… Name should be <
+connector_name>, author is `nakji`, and the version is `0.0.0`.
+
+`types.go`: This defines the maps used by main.go. TopicTypes maps from string to protobuf format structs defined in <
+contract_name>.pb.go. The string keys should be all lowercase, with the format `<author>.<connector_name>.<version>.<
+contract_name>_<event_name>`. The struct values should match the names of structs in `<contract_name>.pb.go`. ABIs maps
+from <contract_name> to the ABI variable in `<contract_name>.abi.go`.
+
+`<connector_name>.go`: This defines the start and parse functions that the connector uses during runtime to start
+receiving and unpacking messages. You can copy the code from `compound.go`, changing the contract name to your specific
+contract.
+
+#### 5. Creating the main executable
+
+Now we just need to create the `main.go` file in `<connector_name>/cmd/<connector_name>/`. This is the main executable
+which
+handles starting and running the connector, as well as receiving user flags for backfill. Copy the code from
+`compound/cmd/main.go`, changing the contract name. You also need to change the network name in the config to the
+chain your contract is deployed on. Congrats! You just built your first connector.
+
+#### 6. Running a connector locally
+
+You should now be able to run your connector locally without any additional steps. From your terminal, go into <
+connector_name>/ and run
+
+```shell
+go mod init
+
+go mod tidy
+```
+
+This automatically creates the `go.mod` and `go.sum` files that tell go to fetch which modules and dependencies your
+program
+depends on.
+
+Next, spin up kafka by running:
+
+```shell
+docker-compose up -d
+```
+
+Now start the connector locally by running the `main.go` file. (If you are running to “connection refused” error, make
+sure docker is running)
+
```

Comment:
I think they also need to run `protoregistry`

Reply:
No they don't, as long as the `env` is set to `dev`